apps/portal/controllers/includes/designer.jag

```diff
@@ -6,7 +6,7 @@
     }
 
     if (userDomain !== (urlDomain || superDomain)) {
-        //response.sendError(401, 'designer access not authorized of this tenant');
+        response.sendError(401, 'designer access not authorized of this tenant');
```

Comment:
The message should be started with uppercase, and the message is grammatically wrong. Please fix this.